src/main/java/org/oasis_eu/portal/controller/StoreController.java

```diff
@@ -146,6 +146,31 @@ public void rate(@PathVariable String appType, @PathVariable String appId, @Requ
         ratingService.rate(appType, appId, rateRequest.rate);
     }
 
+    @RequestMapping(value = "/{appType}/{appId}/organization/disabled", method = GET)
+    public List<UIOrganization> organizationsDisabled(@PathVariable String appType, @PathVariable String appId) {
+        AppstoreHit info = appstoreService.getInfo(appId, CatalogEntryType.valueOf(appType.toUpperCase())); // #152 services can also be installed
+        List<UIOrganization> organizations = organizationService.getMyOrganizations();
+
+        return organizations
+                .stream()
+                .filter(o -> o.isAdmin())
```

Comment:
you can just set one filter for the three conditions, it avoids looping three consecutive times in a collection that may have hundreds of organizations.